Fails to compile with newer firmware.

Given the documentation and a quick perusal of the source, it seems like this plugin is still using the old plugin API. Compile error is below:

> ditto:~/Arduino/Model01-Firmware% make flash
> BOARD_HARDWARE_PATH="/home/bjc/Arduino/hardware" /home/bjc/Arduino/hardware/keyboardio/avr/libraries/Kaleidoscope/bin//kaleidoscope-builder flash
> Building output/Model01-Firmware/Model01-Firmware (0.0.0-gv1.22-26-ga99e-dirty) ...
> /home/bjc/Arduino/hardware/keyboardio/avr/libraries/Kaleidoscope-MacrosOnTheFly/src/Kaleidoscope/FlashOverride.cpp: In member function 'void kaleidoscope::FlashOverride::begin()':
> /home/bjc/Arduino/hardware/keyboardio/avr/libraries/Kaleidoscope-MacrosOnTheFly/src/Kaleidoscope/FlashOverride.cpp:42:16: error: 'class kaleidoscope::Kaleidoscope_' has no member named 'useLoopHook'
>    Kaleidoscope.useLoopHook(loopHook);
>                 ^
> exit status 1
> /home/bjc/Arduino/hardware/keyboardio/avr/build-tools/makefiles//rules.mk:72: recipe for target 'flash' failed
> make: *** [flash] Error 1

--- a/src/Kaleidoscope/MacrosOnTheFly.h
+++ b/src/Kaleidoscope/MacrosOnTheFly.h
@@ -29,7 +29,7 @@
 
 namespace kaleidoscope {
 
-class MacrosOnTheFly : public KaleidoscopePlugin {
+class MacrosOnTheFly : public kaleidoscope::Plugin {
  public:
   MacrosOnTheFly(void);
 

